posthog/settings/ingestion.py

```diff
@@ -11,6 +11,11 @@
 # KEEP IN SYNC WITH plugin-server/src/config/config.ts
 BUFFER_CONVERSION_SECONDS = get_from_env("BUFFER_CONVERSION_SECONDS", default=60, type_cast=int)
 
+# Whether or not random partitioning (i.e. overflow routing) should be allowed.
+# (Enabling this setting this does not cause messages to be randomly
```

Comment:
```suggestion
# (Enabling this setting does not cause messages to be randomly
```

total nit, up to you whether you want to wait on CI again, lol.

Reply:
Eek, thanks for pointing this out!